src/services/schema-form.js

```diff
@@ -96,6 +104,109 @@ angular.module('schemaForm').provider('schemaForm',
     return f;
   };
 
+  var combinePropertyDependencies = function(schema) {
```

Comment:
I don't quite like how this modifies `properties` - going to figure out a way for this to just modify `items`.